src/types.db

```diff
@@ -231,6 +231,12 @@ serial_octets           rx:DERIVE:0:U, tx:DERIVE:0:U
 signal_noise            value:GAUGE:U:0
 signal_power            value:GAUGE:U:0
 signal_quality          value:GAUGE:0:U
+slurm_job_state         value:GAUGE:0:U
+slurm_stats_load        server_thread_count:GAUGE:0:U, agent_thread_count:GAUGE:0:U, agent_queue_size:GAUGE:0:U, dbd_agent_queue_size:GAUGE:0:U
```

Comment:
Ouch, misunderstanding occured.

Types with multiple values are not preferred. all values should go to its own types with type_instance as before.

Most of types should have only one data_source named `value`.
I think, adding of new types with multiple data_sources will be very rare.

Is there is documentation about these statistics values?

Please understand, what threads_count and queue_size is different metrics (that is different kind of load), and they should go to different types, they should not be mixed together.

I will try to fill a table of proposed types.